src/markdown-pages/docs/ecosystem/ecosystem.md

```diff
@@ -0,0 +1,85 @@
+| **Dynamic Data Store**               | The dynamic data store (DSS) is used by Managers and the Galasa framework to ensure that limits set in the CPS configuration are not exceeded. DSS properties change dynamically as tests are run, to show the resources that are currently in use, shared or locked by a test, so that workloads can be limited to avoid throttling. When running in automation, the DSS is shared by every instance of the framework.                                                                 |
+| **Result Archive Store**                      | The result archive store (RAS) stores all elements of a test, including the run log, stored artifacts such as requests, and properties that get changed in the DSS. These elements can be used to help diagnose the cause of any failures encountered as a result of running the test, or to gather information about the test.                                                                                                      |
+| **Credentials Store**                   | tbc |
+| **Certificate Store** | tbc                                                                                  |
+
+</details>
+
+
+<details>
+<summary>Galasa Servers</summary>
+
+|                                   |                                                                                                                          |
+| ------------------------------------- | :----------------------------------------------------------------------------------------------------------------------------------- |
+| **Engine controller**                       | The Engine Controller is responsible for spinning up docker containers to execute individual Galasa automation runs.        |
+| **etcd**                       | Contains the Configuration Property Store (CPS), the Dynamic Status Store (DSS) and the Credentials Store (CREDs). The CPS, DSS are for use with by all users, the CREDs are for automation runs only.        |
+| **couchdb**                       | Contains the Result Archive Store (RAS) which contains the full record of an automated run.        |
+| **Resource Monitor**                       | The Resource Monitor service handles the cleaning up and management of resources in Galasa.        |
+| **Metrics Server**                       | A metrics server to indicate the health of the ecosystem to a Prometheus server. A Prometheus server scrapes and stores metrics from defined endpoints.       |
+| **Bootstrap Server**                       | The Galasa API server which includes the bootstrap       |
+| **Framework API Server**                       | The API server is an API mediation layer which sits between the Galasa framework and Galasa services, such as the properties services in the CPS. The API server acts as a central point from which to control the Galasa ecosystem.        |
```

Comment:
There isnt a Framework API and Manager API server, just an API server. Its an endpoint allowing for things like test submissions